io.github.qnapi.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<component type="desktop">
+	<id>io.github.qnapi</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-2.0+</project_license>
+	<name>QNapi</name>
+	<summary>A software for downloading and automatic matching subtitles to movies</summary>
+	<description>
+		<p>A software for downloading and automatic matching subtitles to movies. Works on Linux, OSX and Windows.</p>
+	</description>
+	<launchable type="desktop-id">qnapi.desktop</launchable>
+	<url type="homepage">https://qnapi.github.io/</url>
+	<screenshots>
+		<screenshot type="default">
+			<image>https://qnapi.github.io/images/screenshots/0.1.7/lin05.png</image>
+		</screenshot>
+	</screenshots>
+	<content_rating type="oars-1.1">
+		<content_attribute id="social-chat">intense</content_attribute>

```

Comment:
Not sure why it has social chat to intense?

Reply:
It's was pretty hard to select something because it an app to download subtitles. Subtitles can have some intense content. qBittorrent don't have that so maybe it's a bad choice.

Reply:
I would not have this. Thanks for thinking hard about it though!

Reply:
Yeah, let's ditch it.

Reply:
OK. So if I understand currently, I should not remove that.